File: test/fixtures/langchain-repo/lc_openai.ts
import { ChatOpenAI } from "@langchain/openai";
import { SystemMessage, HumanMessage } from "@langchain/core/messages";

const llm = new ChatOpenAI({ model: "gpt-4o" });

export async function ask(q: string) {
  return llm.invoke([
    new SystemMessage("You are a meticulous assistant."),
    new HumanMessage("What is the capital of France?"),
  ]);
}

File: test/fixtures/langchain-repo/lc_openai.py
from langchain_openai import ChatOpenAI
from langchain_core.messages import SystemMessage, HumanMessage

llm = ChatOpenAI(model="gpt-4o", temperature=0)

def ask(q):
    return llm.invoke([
        SystemMessage(content="You are a meticulous assistant."),
        HumanMessage(content="What is the capital of France?"),
    ])
